Views/Home/Index.cshtml

```diff
@@ -1,7 +1,51 @@
 ﻿@using Widgets.Data.Models
+@using Widgets.Services
+
 @{
     ViewData["Title"] = "Home Page";
 }
+<head>
```

Comment:
The head element located within the body of the html document is malformed html.  The head element should be at the sibling level and appear before the body element.

Reply:
added in correct HTML tags, sorry I overlooked those being missing.

Reply:
Please locate the _Layout.cshtml file in the solution.  You will notice the root level elements here.  The framework loads all other html elements created in the app inside the body element found in this file.  You will also find the head element in this file, and this would be the well-formed area to locate your styles.

Reply:
Ah the pages act as component files and _layout is the master file.